src/systems/ec/implementations/vanilla/packages/ec-component-list/ec-component-list.scss

```diff
@@ -0,0 +1,127 @@
+/**
+ * List component
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+/**
+ * @define ordered-list
+ */
+@mixin ecl-ordered-list(
+  $text-color: $ecl-color-grey-100,
+  $divider-color: $ecl-color-grey-15
+) {
+  .ecl-ordered-list {
+    color: $text-color;
+    font: $ecl-font-prolonged-m;
+    margin: 0;
+    // Approximately centered bullet
+    padding-left: calc((#{$ecl-spacing-s} * 2) + 0.3em);
+  }
+
+  .ecl-ordered-list .ecl-ordered-list {
+    margin-top: $ecl-spacing-xs;
+  }
+
+  .ecl-ordered-list__item {
+    margin-top: $ecl-spacing-xs;
+
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+/**
+ * @define unordered-list
+ */
+@mixin ecl-unordered-list(
+  $text-color: $ecl-color-grey-100,
+  $divider-color: $ecl-color-grey-15
+) {
+  .ecl-unordered-list {
+    color: $text-color;
+    font: $ecl-font-prolonged-m;
+    margin: 0;
+    // Approximately centered bullet
+    padding-left: calc((#{$ecl-spacing-s} * 2) + 0.3em);
+  }
+
+  .ecl-unordered-list .ecl-unordered-list {
+    margin-top: $ecl-spacing-xs;
```

Comment:
wondering... if the same top spacing is used between list items AND within an item, between the label and the nested list, how do you know the nested list is not another list item (not related to the label)?

Reply:
well, with the indentation